docs/concepts/lockup/02-stream-shapes.mdx

```diff
@@ -9,7 +9,7 @@ import FunctionPlot from "@site/src/components/FunctionPlot";
 :::note
 
 - The code used to generate the gas benchmarks for the different stream curves can be found
-  [here](https://github.com/sablier-labs/examples/tree/shapes-benchmark).
+  [here](https://github.com/sablier-labs/benchmarks).
```

Comment:
This should not be changed. I forgot that this requires closing https://github.com/sablier-labs/benchmarks/issues/1 first.